src/pages/events/index.tsx

```diff
@@ -0,0 +1,61 @@
+import { EventType } from '@/lib/types';
+
+import Events from '@/components/upcoming/Events';
+
+import { NextPage } from 'next';
+
+const events: EventType[] = [
+  {
+    id: 'id1',
+    name: 'Hackoona Matata',
+    location: 'New Delhi',
+    date: new Date(2023, 8, 10),
+    link: '#',
+    image:
+      'https://source.unsplash.com/random?300x300?fest,hackathon,coding,tech',
+  },
+  {
+    id: 'id2',
+    name: 'HackCBS',
+    logo: 'https://hackcbs.tech/assets/img/logo_final.png',
+    location: 'New Delhi',
+    date: new Date(2023, 8, 10),
+    link: '#',
+    image:
+      'https://source.unsplash.com/random?300x300?hackathon,technology, conferences',
+  },
+  {
+    id: 'id3',
+    name: 'Hackerstellar',
+    location: 'New Delhi',
+    date: new Date(2023, 8, 10),
+    link: '#',
+    image: 'https://source.unsplash.com/random?300x300?hack,computers',
+  },
+  {
+    id: 'id4',
+    name: 'HackCBS',
+    location: 'New Delhi',
+    date: new Date(2023, 8, 10),
```

Comment:
It will be good that you pass the time zone, reason: people from outside of india may see wrong date

Reply:
@Chandraprakash-Darji could you elaborate it? Like what'd be recommend way to pass the timezone?